templates/back-office/curbside-base.hypr

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html> 
+<html>
+    <head>
+        <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+        {% include "modules/json-required-for-hyprlive" %}
+        {% require_script "modules/login-links" %}
+        {% require_script "modules/cart-monitor" %}
+        {% include "modules/trailing-scripts" %}
+    	    {% cms_resources %}
+        {% block stylesheets %}
+        <link rel="stylesheet" href="{% make_url "stylesheet" "/stylesheets/storefront.less" %}" title="default" />
+        {% endblock stylesheets %}
+        <style type="text/css">{% inline_style "back-office.css" %}</style>
+    </head>
+    <body>   
+        {% block body-content %}
```

Comment:
Probably need a header / footer block here. And then break header and footer into separate templates